.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
           "console": "integratedTerminal",
           "internalConsoleOptions": "openOnFirstSessionStart",
           "env": {
-            "TEST_FILES": "BuildTest"
+            "TEST_FILES": "appxTest"
```

Comment:
Why does this need to be modified? Was it forgotten to be deleted?

Reply:
Forgot to be deleted. I'll update the PR